x-pack/test_serverless/functional/test_suites/observability/dataset_quality/dataset_quality_details.ts

```diff
@@ -177,7 +177,9 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         expect(parseInt(degradedDocs, 10)).to.be(1);
         expect(parseInt(services, 10)).to.be(3);
         expect(parseInt(hosts, 10)).to.be(52);
-        expect(parseInt(size, 10)).to.be.greaterThan(0);
+        // metering stats API is cached for 30seconds, waiting for the exact value is not optimal in this case
+        // rather we can just check if any value is present
+        expect(size).to.be.ok();
```

Comment:
> [!NOTE]
> The [.ok](https://www.chaijs.com/api/bdd/#method_ok) method is not recommended and asserts for truthiness, not existence, so it should fail too when the size is 0.
> If we cannot rely on concrete criteria for asserting the value of `size`, it makes it more difficult to test this part as we cannot reliably know if we get the cached value.
> I cannot suggest a better alternative, unfortunately, just raising that this assertion might trigger the same issue as before.

Reply:
> so it should fail too when the size is 0.

`size` is being returned as an string. As long as something is returned in the UI we should be fine.

> we cannot reliably know if we get the cached value

This is the main problem that's why I'm proposing not to care about the value at least from a UI perspective and just  assert we have a value there. For the accuracy of the value I prefer we rely on the API tests. wdyt?